pkg/shell/shell-modals.jsx

```diff
@@ -139,11 +139,27 @@
                     <Divider />
                     <MenuContent>
                         <MenuList>
-                            {Object.keys(manifest.locales || { })
-                                    .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()))
-                                    .map(key => {
-                                        return <MenuItem itemId={key} key={key} data-value={key}>{manifest.locales[key]}</MenuItem>;
-                                    })}
+                            {
+                                (() => {
+                                    const filteredLocales = Object.keys(manifest.locales || {})
+                                            .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()));
+
+                                    if (filteredLocales.length === 0) {
+                                        return (
+                                            <MenuItem isDisabled>
+                                                {_("No languages match")}
+                                            </MenuItem>
+                                        );
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18689-20230427-142709-3543de05-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)